src/config/default.json

```
@@ -25,6 +25,15 @@
         "md": 4,
         "lg": 8
     },
+    "boxShadow": {
+        "dark": {
+            "x": 5,
+            "y": 5,
+            "spread": 3,
+            "color": "rgba(0, 0, 0, 0.5)",
+            "blur": 5
+        }
+    },
```

This will lead to the UI showing 4 different tokens (the plugin basically checks for a `value` prop in a token, if it finds that, it is understood to be a single token rather than multiple). An exception are typography tokens, which have some logic involved due to wanting to support old no-value tokens. But I'd say for new tokens like `boxShadow` we can force the new syntax.

So, adding the `value` prop will make sure that the token is displayed as a single token.

```suggestion
    "boxShadow": {
        "dark": {
          "value": {
            "x": 5,
            "y": 5,
            "spread": 3,
            "color": "rgba(0, 0, 0, 0.5)",
            "blur": 5
          }
        }
    },
```

---

👍 another good catch!